### PR TITLE
CASMINST-3717-csm-1.2 - update cray bss update param

### PR DESCRIPTION
**Pull request by dle-hpe**
# Description

- update cray bss update parameter for [operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md](https://github.com/Cray-HPE/docs-csm/blob/release/1.2/operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md)
# Checklist Before Merging

- [ ] If I added any command snippets, the steps they belong to follow the prompt conventions (see [example][1]).
- [ ] If I added a new directory, I also updated `.github/CODEOWNERS` with the corresponding team in [Cray-HPE][2].
- [ ] My commits or Pull-Request Title contain my JIRA information, or I don't have a JIRA.

[1]: https://github.com/Cray-HPE/docs-csm/blob/main/introduction/documentation_conventions.md#using-prompts
[2]: https://github.com/Cray-HPE/teams

### CODE REVIEW

File: operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md [hunk 1]
```diff
@@ -332,7 +332,7 @@ This step updates the entries in BSS for the NCNs to use the new images.
                 kernel=$(cat bss_$xname.json | jq '.[]  .kernel')
                 initrd=$(cat bss_$xname.json | jq '.[]  .initrd')
                 params=$(cat bss_$xname.json | jq '.[]  .params')
-                cray bss bootparameters update --initrd $initrd --kernel $kernel --params $params --name $xname --format json
+                cray bss bootparameters update --initrd $initrd --kernel $kernel --params $params --host $xname --format json
```
Comment (spillerc-hpe) [inline operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md:335]: I think this is wrong and the option should be `--hosts`

**Review by spillerc-hpe — CHANGES_REQUESTED** [review]

File: operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md [hunk 2]
```diff
@@ -350,7 +350,7 @@ This step updates the entries in BSS for the NCNs to use the new images.
...
```
Comment (spillerc-hpe) [inline operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md:353]: This should also be `--hosts`

File: operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md (hunk 1 above)
Comment (dle-hpe) [reply operations/security_and_authentication/Change_NCN_Image_Root_Password_and_SSH_Keys.md]: updates my type-o

**Review by spillerc-hpe — APPROVED** [review]

**Review by heemstra — CHANGES_REQUESTED** [review]
pretty sure "$params" needs to be quoted in both snippets

**Comment by dle-hpe** [comment]
> pretty sure "$params" needs to be quoted in both snippets

i think i got it...

**Review by heemstra — APPROVED** [review]

**Pull request merged**